src/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ]  Great job so far 👏🏻 , However, Getters and Setters for @book and @person instance variables should be present and Ruby offers you a nice way to create your getters and setters! you can declare them by using :

` attr_reader,  attr_writer`,  # Or you can make it even shorter by using: `attr_accessor `

 Actually, you only have a getter method `attr_reader`. In order to make sure the  Setters for @book and @person are present as well, let's add this line.

```ruby
  attr_accessor :date, :book, :person
```
Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor%20#%20*:name%20and%20:age%20instance%20variables%20are%20the%20only%20ones%20that%20should%20be%20able%20to%20be%20modified%20that's%20why%20they%20are%20defined%20as%20attr_accessors%20(here%20both%20Getters%20and%20Setters%20are%20defined)) to the example in a previous lesson: